examples/usercall-extension/test.sh

```diff
@@ -1,17 +1,24 @@
 set -e
 
+APPNAME=app
```

Comment:
Are the changes to this file necessary?

Instead of the changes to this file, I do think this example needs to be updated to support `AsyncStream/AsyncListener`.